recipes/google-cloud-cpp-core/build.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -euo pipefail
+
+export OPENSSL_ROOT_DIR=$PREFIX
+
+if [[ "${target_platform}" == osx-* ]]; then
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+  CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+echo "$(date -u '+%Y-%m-%dT%H:%M:%SZ'): Building __common__ features..."
+cmake ${CMAKE_ARGS} \
+    -GNinja -S . -B build_common \
+    -DGOOGLE_CLOUD_CPP_ENABLE=__common__ \
+    -DBUILD_TESTING=OFF \
+    -DBUILD_SHARED_LIBS=ON \
+    -DOPENSSL_ROOT_DIR=$PREFIX \
+    -DCMAKE_BUILD_TYPE=release \
+    -DCMAKE_CXX_STANDARD=17 \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    -DProtobuf_PROTOC_EXECUTABLE=$BUILD_PREFIX/bin/protoc \
+    -DGOOGLE_CLOUD_CPP_GRPC_PLUGIN_EXECUTABLE=$BUILD_PREFIX/bin/grpc_cpp_plugin \
+    -DGOOGLE_CLOUD_CPP_ENABLE_WERROR=OFF
+
+cmake --build build_common
+cmake --install build_common --prefix stage
+echo "$(date -u '+%Y-%m-%dT%H:%M:%SZ'): DONE - Building __common__ features"
+
+### This loop would go in the google-cloud-cpp-core-feedstock
+### Other feedstocks may have similar loops, for example:
+### feedstock name        | list
+### google-cloud-cpp-core | __common__ bigtable oauth2
+###                       | (iam pubsub policytroubleshooter) spanner storage
```

Comment:
I don't understand the conjunctive here. This loop would then be in the `google-cloud-cpp-core` feedstock. Also, what do the brackets under `list` mean?